src/fluree/db/ledger/mutable.clj

```diff
@@ -127,7 +126,7 @@
   (<?? (purge-flakes conn "fluree" "test" {:purge [422212465065991]}))
 
   (->> (<?? (storage/storage-read conn "fluree_test_block_000000000000004:v2"))
-       (serdeproto/-deserialize-block (:serializer conn)))
+       ( fluree.db.serde.protocol/-deserialize-block (:serializer conn)))
```

Comment:
Same question as above here. Also there's a space after the opening parens that shouldn't be there.

Reply:
We/I turned off linting against comments since the linter(s) complained about various items (e.g., unnecessary let, unused bindings, ...)  

The above code is in the comment section.  The other option would be to delete these comment lines?

Reply:
Ohhh... I see. And so it still complains about unused imports then huh?

Reply:
Yes.  I can delete code; but, I have no idea if anyone on our team still uses any of the code in the comments.  When I leave code in the comments... I am generally looking to add a unit test.

Reply:
Since that code is intended for REPL evaluation (I assume), we may want to just move the requires into the top of the `(comment ...)` form like this:

```clojure
(comment

  (require '[fluree.db.serde.protocol :as serdeproto])
  ...
  (serdeproto/-deserialize-block (:serializer conn)))
```